src/js/cypress/Widget.ts

```diff
@@ -14,15 +14,15 @@ export default class Widget {
     }
 
     waitForVisible(timeout?: BrowserTimeout) {
-        return this.browser.waitForVisible(this.selector, timeout);
```

Comment:
The `return` statement is redundant here as we should never use `.then()` after `waitForVisible()`. It also prevents us from redefining this method in subclasses as we need to keep the method return type there, which is not always possible.